.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
```

Comment:
I think this should go to gitignore

Reply:
I fixed the linting issues, but im not sure about this live server thing, is it from my branch?